service/static/js/index.js

```diff
@@ -67,7 +84,26 @@ $(function () {
     })
 
     ajax.done(function(res){
-      update_wishlist_form_data(res["data"])
+      $("#search_results").empty();
+      $("#search_results").append('<table class="table-striped" cellpadding="10">');
+      var header = '<tr>'
+      header += '<th style="width:10%">ID</th>'
+      header += '<th style="width:30%">Name</th>'
+      header += '<th style="width:10%">User_ID</th>'
+      header += '<th style="width:50%">Products</th></tr>'
+      $("#search_results").append(header);
+      res = res["data"]
+      var row = "<tr><td>"+res.id+"</td><td>"+res.name+"</td><td>"+res.user_id+"</td><td>"+res.products+"</td></tr>";
+      $("#search_results").append(row);
+
+
+      $("#search_results").append('</table>');
+
+      // copy the first result to the form
+      update_wishlist_form_data(res)
```

Comment:
Please make sure to use `;` at the end of every line. JS technically can manage without them, but it can lead to side effects and unexpected behavior in some cases.